includes/Customer.php

```diff
@@ -248,6 +256,29 @@ public static function connect( $path ) {
 		return json_decode( wp_remote_retrieve_body( $response ) );
 	}
 
+	/**
+	 * Check if maybe_refresh_token will retrieve a valid token
+	 * matching logic in Bluehost/AccessToken class for maybe_refresh_token
+	 *
+	 * @return bool
+	 */
+	private static function can_access_token() {
+
+		// Is this is an ajax request
+		if ( defined( 'DOING_AJAX' ) && DOING_AJAX ) {
+			// Unable to acquire AccessToken on ajax requests
+			return false;
+		}
+
+		// Does user have permissions or is this cron?
+		if ( ! ( current_user_can( 'manage_options' ) || wp_doing_cron() ) ) {
+			// Unable to acquire AccessToken either without user priveledges or for cron events
+			return false;
+		}
+
+		return true;
```

Comment:
What about just something like this: 

```suggestion
		return ( current_user_can( 'manage_options' ) || wp_doing_cron() );
```

Reply:
I like this, and am happy to bring it in